packages/sdk/src/utils/load.ts

```diff
@@ -1,11 +1,16 @@
-/** 
- * This is a wrapper around `require` 
+import fs from 'fs'
+
+/**
+ * This is a wrapper around `require`
  * This is to avoid eval and hide require away from bundlers
  */
 export function load<T>(id: string): T {
   try {
     return require(id) as T
   } catch (e) {
+    const { size } = fs.statSync(id)
+
+    console.log('BINARY_SIZE', size)
```

Comment:
Looks unrelated to types discovery, what is it for?

Reply:
Oh yes, that was a debug for tests not passing. Good catch!